<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\informacao_produto\add_version_to_product_dialog.ui -->
<ui version="4.0">
 <class>AddVersionToProductDialog</class>
 <widget class="QDialog" name="AddVersionToProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Nova Versão ao Produto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="versionInfoGroupBox">
     <property name="title">
      <string>Informações da Versão</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nomeVersaoLabel">
          <property name="text">
           <string>Nome da Versão:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nomeVersaoLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="versaoLabel">
          <property name="text">
           <string>Número da Versão:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="versaoLineEdit">
          <property name="placeholderText">
           <string>Ex: 1.0, 2ª Edição</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="tipoVersaoLabel">
          <property name="text">
           <string>Tipo de Versão:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tipoVersaoComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="subtipoProdutoLabel">
          <property name="text">
           <string>Subtipo de Produto:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="subtipoProdutoComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="loteLabel">
          <property name="text">
           <string>Lote:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="loteComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="orgaoProdutorLabel">
          <property name="text">
           <string>Órgão Produtor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="orgaoProdutorLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="palavrasChaveLabel">
          <property name="text">
           <string>Palavras-chave:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="palavrasChaveLineEdit">
          <property name="placeholderText">
           <string>Separadas por vírgula</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="dataCriacaoLabel">
          <property name="text">
           <string>Data de Criação:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dataCriacaoDateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="dataEdicaoLabel">
          <property name="text">
           <string>Data de Edição:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dataEdicaoDateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="descricaoVersaoLabel">
        <property name="text">
         <string>Descrição da Versão:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="descricaoVersaoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="metadadoLabel">
        <property name="text">
         <string>Metadados (JSON):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="metadadoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fileGroupBox">
     <property name="title">
      <string>Arquivos</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="tipoArquivoLabel">
          <property name="text">
           <string>Tipo de Arquivo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tipoArquivoComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="crsLabel">
          <property name="text">
           <string>CRS Original:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="crsLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="descricaoArquivoLabel">
        <property name="text">
         <string>Descrição do Arquivo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="descricaoArquivoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="addFileButton">
          <property name="text">
           <string>Adicionar Arquivo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeFileButton">
          <property name="text">
           <string>Remover Arquivo Selecionado</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="filesTable">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroupBox">
     <property name="title">
      <string>Progresso do Upload</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Pronto para upload.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>